bin/all_sky_search/pycbc_calculate_dq

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+""" Ingest and process DQ data
```

Comment:
Can this docstring be made a bit more descriptive? What kind of DQ data does this script take as input, and what does it do when it "processes" it?